sdk/identity/Azure.Identity/tests/samples/CustomCredentialSnippets.cs

```diff
@@ -68,7 +68,7 @@ public void ConfidentialClientCredentialUsage()
             #endregion
         }
 
-        #region Snippet:OnBehalfOfCredential
+        //#region Snippet:OnBehalfOfCredential
```

Comment:
Nit: given it's in source control, any reason to not just remove this region declaration?

Reply:
This is actually the state that this snippet file is in right now in main: https://github.com/Azure/azure-sdk-for-net/blob/5a29f2e99c52728ea8aaa646f1101761509bc57e/sdk/identity/Azure.Identity/tests/samples/CustomCredentialSnippets.cs#L71. I left it originally since it's a code snippet many have picked up and we might have to support them. It possibly could be removed but I'm not going to do it as part of this PR.